content/blog/2021/containerqueries.md

```diff
@@ -88,6 +96,19 @@ as well.
 proposal. `Inline-size` seemed to satisfy more use-cases, so is being developed
 first.)
 
+{% callout 'note', update %}
+
+- The `size` containment type, which queries in both the inline and
```

Comment:
I think this could be more clear that `size` and `inline-size` are both well supported, but containing only the `block-size` isn't possible.

Reply:
Maybe worth noting that `layout` containment has been removed from the spec, but is still applied in some browsers that haven't updated their implementation

Reply:
I think I cleared this up in https://github.com/oddbird/oddleventy/pull/852/commits/5ccd06ed54d5bc218558966c4678630e24151188